include/mpact/Transforms/Passes.h

```diff
@@ -0,0 +1,22 @@
+//===------------------------------------------------------------*- C++ -*-===//
+//
+// This file is licensed under the Apache License v2.0 with LLVM Exceptions.
```

Comment:
use your original header comment

 Part of the MPACT Project, under the Apache License v2.0 with LLVM

here and elsewhere, I liked that one!